AdminWebsite/AdminWebsite/ClientApp/src/app/services/clients/api-client.ts

```diff
@@ -2650,6 +2650,53 @@ export interface IEndpointRequest {
     defence_advocate_username?: string | undefined;
 }
 
+export enum LinkedParticipantType {
+    Interpreter = 'Interpreter'
+}
+
+export class LinkedParticipantRequest implements ILinkedParticipantRequest {
```

Comment:
Classes like this in TS aren't required, the interface alone is enough. All of this code will get chucked out by the compiler, so I wouldn't bother with it. Anywhere you're using the class, just use the interface instead 👍 Interfaces and classes in TS aren't the same as in C#, they're really just tools for type safety rather than code that actually gets 'built'

Reply:
this file (api-client.ts) is auto generated